omod-2.4/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_4/FrontendJsonConfigController2_4.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_4;
```

Comment:
Is there any reason why you put this in 2.4?

Reply:
I assumed that new endpoints must be in the latest omod version, ie 2.4 at the moment

Reply:
Our convention is that the location of your class is determined by the lowest platform version of OpenMRS required for your functionality.

Reply:
I think i need to add it to 2.2 then

Reply:
Does not have any functionality that requires 2.2 and above but not below?

Reply:
It actually depends on O3 whose platform minimum is 2.2 
I think this feature is not necessary to users who are not running O3

Reply:
O3 requiring platform 2.2 is knowledge that this module should not be aware of. 😊

Reply:
I will add the logic to 1.8. them

Reply:
i have added these changes to 1.9 then